src/pages/components/TabView.qml

```diff
@@ -39,12 +42,15 @@ SilicaGridView {
     delegate: TabItem {
         id: tabItem
 
+        readonly property bool largeLeftMargin: index % columns === 0
+        readonly property bool largeRightMargin: columns > 1 ? index % columns === (columns - 1) : true
```

Comment:
Thanks @kananoja will do suggested changes. firstColumn and lastColumn are better names.